shipyard_blueprints/sftp/shipyard_sftp/cli/download_file.py

```diff
@@ -64,9 +64,8 @@ def main():
             transport.default_window_size = DEFAULT_WINDOW_SIZE
             transport.packetizer.REKEY_BYTES = DEFAULT_REKEY_BYTES
             transport.packetizer.REKEY_PACKETS = DEFAULT_REKEY_PACKETS
-            transport.connect(None, args.username, args.password)
-
             connection_args["transport"] = transport
```

Comment:
does this need to be connection_args.get("transport") instead to ensure that it defaults to `None`? Or does it have a default value in the CLI?